lua-http-parser.c

```diff
@@ -473,7 +620,6 @@ static int lhp_is_function(lua_State* L) {
  * can yield without having to apply the CoCo patch to Lua. */
 static const char* lhp_execute_lua =
     "local c_execute, is_function = ...\n"
-    "local type = type\n"
```

Comment:
Please leave this in since it is supposed to improve the performance (at least in Lua 5.2 and earlier).

Reply:
Code does not use `type` function.
But maybe its better use it because LuaJIT can impement it faseter than call
C function i think

Reply:
Your right! My mistake, I forgot that we are now using lhp_is_function() where as previously I think we were using "function"==type().